### PR TITLE
convert readme to reStructuredText for PyPI

### PR DESCRIPTION
**Pull request by graingert**
Here we use reStructuredText rather than markdown because it renders as HTML on PyPI.

Similar to https://github.com/pytorch/vision/pull/31

### CODE REVIEW

**Comment by apaszke** [comment]
`:width:` doesn't seem to work here (logo takes up 100%) and the other images are missing

**Comment by graingert** [comment]
@apaszke hmm that's a pain, will have to try fiddling around with raw

**Comment by apaszke** [comment]
That shouldn't be too hard, right? We have it in raw html already

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+-  `Installation <#installation>`__
+-  `Binaries <#binaries>`__
+-  `From source <#from-source>`__
+-  `Getting Started <#getting-started>`__
+-  `Communication <#communication>`__
+-  `Releases and Contributing <#releases-and-contributing>`__
+-  `The Team <#the-team>`__
+
++-----------+---------------------+---------------------+
+| Python    | **``Linux CPU``**   | **``Linux GPU``**   |
```
Comment (lukeyeager) [inline README.rst]: Backticks don't seem to work the same in RST

Comment (graingert) [inline README.rst]: yeah, I just assumed pandoc was correct here

**Comment by apaszke** [comment]
I'm closing this for now, due to inactivity and conflicts. You can open another PR once the issues with logo are resolved. Thanks!

**Comment by graingert** [comment]
Fair looks like github needs to tweak their reStructuredText renderer

On 17 Feb 2017 18:09, "Adam Paszke" <notifications@github.com> wrote:

Closed #522 <https://github.com/pytorch/pytorch/pull/522>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/pytorch/pytorch/pull/522#event-967345001>, or mute the
thread

**Pull request closed without merging**